Apps/PatientDataAccess/src/Api/IPatientApi.cs

```diff
@@ -36,46 +31,45 @@ internal enum DonorStatus
         Pending,
     }
 
+    internal enum HealthDataCategory
+    {
+        Laboratory,
+        COVID19Laboratory,
+        ClinicalDocument,
+        DiagnosticImaging,
+    }
+
+    internal enum DiStatus
+    {
+        Scheduled,
+        InProgress,
+        Finalized,
+        Pending,
+        Completed,
+        Amended,
+    }
```

Comment:
Isn't this the 3rd enum for this status?  I thought DiagnosticImagingExamStatus was the one that corresponded to the external model.

Reply:
Yes. This is a part I struggled with, they belong to each of the following domains:

PHSA -> PatientDataAccess -> Patient